Library/Homebrew/test/exceptions_spec.rb

```diff
@@ -202,33 +202,33 @@ class Baz < Formula; end
   describe ChecksumMismatchError do
     subject { described_class.new("/file.tar.gz", hash1, hash2) }
 
-    let(:hash1) { double(hash_type: "sha256", to_s: "deadbeef") }
-    let(:hash2) { double(hash_type: "sha256", to_s: "deadcafe") }
+    let(:hash1) { instance_double(Checksum, to_s: "deadbeef") }
+    let(:hash2) { instance_double(Checksum, to_s: "deadcafe") }
```

Comment:
i think this is the correct `instance_double`, but it doesn't recognize `hash_type`